Test_design.md

```diff
@@ -0,0 +1,64 @@
+# Test design for selected functionalities of the website pracuj.pl
+
+## Logowanie
+
+### Przez e-mail

```

Comment:
Uwaga ogólna do wszystkich nazw testów:

Nazwa testy powinna odzwierciedlać to co test robi, w tym wypadku powinno być coś w rodzaju "Valid e-mail login"